### DISCUSSION TITLE
integration into landscape

### DISCUSSION THREAD

**Discussion by wibed** [plausible/analytics#4661] [Self-Hosted Support] [answered]
i am looking for a way to track user behaviour such as in google analytics. come upon your repoand have session management within the backend already backed in.

how do i integrate with plausible to share session data with the analytics framework?

**Comment by ruslandoga** [comment, marked as answer]
👋 @wibed 

Plausible maintains own user sessions by using a hash of daily_salt, website_domain, client ip_address, and client user_agent and the session id. Please see "How we count unique users without cookies" at https://plausible.io/data-policy. So, out of the box, it's not designed to integrate with existing backend sessions. However, some users have reported they were successful in using custom event properties to do that.

**Reply by wibed** [reply]
on a side note:
if id like to map session data to plausible analytics would that be in breach of contract?
gdpr or whatever

**Reply by ruslandoga** [reply]
I don't know :)
